sly.el

```diff
@@ -1674,10 +1678,6 @@ EVAL'd by Lisp."
   (and (>= (buffer-size) 6)
        (>= (- (buffer-size) 6) (sly-net-decode-length))))
 
-(defun sly-run-when-idle (function &rest args)
```

Comment:
I like this: these one-off half-decent abstractions (in the words of the former SLIME maintainer) aren't really useful and complicate reading.  I welcome this.